editor.planx.uk/src/@planx/components/FileUploadAndLabel/Public.tsx

```diff
@@ -188,10 +189,6 @@ function Component(props: Props) {
   return (
     <Card
       handleSubmit={props.hideDropZone ? props.handleSubmit : validateAndSubmit}
-      isValid={
-        props.hideDropZone ||
-        slots.every((slot) => slot.url && slot.status === "success")
-      }
```

Comment:
@ianjon3s Having a condition here on the `isValid` prop means that the "Continue" button is disabled, which goes against our general a11y principles. 

Did we ever discuss this one, or is removing this here the right way to go?

Reply:
I've actually removed it in this PR - added to checklist and marked as done!